msal/src/test/java/com/microsoft/identity/client/e2e/tests/network/nativeauth/SignUpTest.kt

```diff
@@ -48,6 +48,29 @@ class SignUpTest : NativeAuthPublicClientApplicationAbstractTest() {
         Dispatchers.setMain(testDispatcher)
     }
 
+    private fun <T> retryOperation(
+        maxRetries: Int = 3,
+        onFailure: () -> Unit = { Assert.fail() },
+        block: () -> T
+    ): T? {
+        var retryCount = 0
+        var shouldRetry = true
+
+        while (shouldRetry) {
+            try {
+                return block()
```

Comment:
Is this return needed? I.e. the entire use of generics `T`. We only need the `block` code to execute, right?

Reply:
Remove the T? and return null.